simple_clock/app.js

```diff
@@ -1,3 +1,237 @@
+            var date = now.getDate();
+            var day = now.getDay();
+            var timezone = now.getTimezoneOffset() / 60 * -1;
+            
+            var hour = now.getHours();
+            var minute = now.getMinutes();
+            var second = now.getSeconds();
+            
+            switch(day) {
+                case 0:
+                day = 'Sunday';
+                break;
+                case 1:
+                day = 'Monday';
+                break;
+                case 2:
+                day = 'Tuesday';
+                break;
+                case 3:
+                day = 'Wednesday';
+                break;
+                case 4:
+                day = 'Thursday';
+                break;
+                case 5:
+                day = 'Friday';
+                break;
+                case 6:
+                day = 'Saturday';
+                break;
+            }
+            if(timezone >= 0) {
+                timezone = '+' + timezone;
+            }
+    
+        var timeValue = year + '/' + month + '/' + date + ' ' + day; 
+        timeValue += (hour>= 12) ? "下午 " : "上午 "; // if condition ? true : false
+        timeValue += ((hour > 12) ? hour - 12 : hour) + " 點:";
+        timeValue += ((minute < 10) ? " 0" : " ") + minute + " 分:";
+        timeValue += ((second < 10) ? " 0" : " ") + second + " 秒";
+        timeValue += ' UTC' + timezone;
+        clock.innerHTML = timeValue;
+        }
+        /*
+        update() {
+            var now = new Date();
+            var year = now.getFullYear();
+            var month = now.getMonth() + 1;
+            var date = now.getDate();
+            var hour = now.getHours();
+            var minute = now.getMinutes();
+            var second = now.getSeconds();
+            var day = now.getDay();
+            var timezone = now.getTimezoneOffset() / 60 * -1;
+            if(minute < 10) {
+                minute = '0' + minute;
+            }
+            if(second < 10) {
+                second = '0' + second;
+            }
+            switch(day) {
+                case 0:
+                day = 'Sunday';
+                break;
+                case 1:
+                day = 'Monday';
+                break;
+                case 2:
+                day = 'Tuesday';
+                break;
+                case 3:
+                day = 'Wednesday';
+                break;
+                case 4:
+                day = 'Thursday';
+                break;
+                case 5:
+                day = 'Friday';
+                break;
+                case 6:
+                day = 'Saturday';
+                break;
+            }
+            if(timezone >= 0) {
+                timezone = '+' + timezone;
+            }
+            document.getElementById('date').innerHTML = year + '/' + month + '/' + date + ' ' + day;
+            document.getElementById('time').innerHTML = hour + ':' + minute + ':' + second + ' UTC' + timezone;
+        }*/
```

Comment:
You should not commit commented code into VCS. You can find it in the history anyway.